ansible_deployer/command_line.py

```diff
@@ -377,7 +391,8 @@ def run_task(config: dict, options: dict, inventory: str):
         sys.exit(70)
     else:
         for playbook in playbooks:
-            command = ["ansible-playbook", "-i", inventory, playbook]
+            command = ["ansible-playbook", "-l", options["limit"], "-i", inventory, playbook] \
+            if options["limit"] else ["ansible-playbook", "-i", inventory, playbook]
```

Comment:
looks difficult to read and when adding tag will even be more complicated, maybe:
```
command = ["ansible-playbook", "-i", inventory, playbook]
if options["limit"]:
 command.append(-l)
 command.append(...)
 ```